test/liquid_handle_dispense_test.py

```diff
@@ -92,22 +92,18 @@ def test_source_configuration(self):
                 )
 
     def test_destination_configuration(self):
+        # Destinations must match the number of chips specified in the source tuple
+        source = [(self.src_tube_1, 2), (self.src_tube_2, 1)]
         test_input_types = [
-            (does_not_raise(), self.flat.well(0)),
-            (does_not_raise(), self.flat.wells_from(0, 2)),
-            (does_not_raise(), WellGroup(self.flat.well(0))),
+            (does_not_raise(), [self.flat.wells_from(0, 2), self.flat.well(3)]),
+            (pytest.raises(ValueError), [self.flat.wells_from(0, 2)]),
             (
                 does_not_raise(),
-                [WellGroup(self.flat.well(0)), WellGroup(self.flat.well(1))],
+                [[self.flat.well(0), self.flat.well(1)], self.flat.well(0)],
             ),
-            (
-                pytest.raises(ValueError),
-                [self.flat.well(0), self.flat.well(1), self.flat.well(0)],
-            ),
-            (does_not_raise(), [self.flat.well(0), self.flat.well(1)]),
+            (pytest.raises(ValueError), [self.flat.well(0), self.flat.well(1)]),
```

Comment:
related to other comment. maybe im forgetting a part of the conversation we had but i dont think we need the cihps to match the number of destination wells per source(because we dont know that each chip is going to dispense to exactly one location - the tempest software takes care of that decision making)